### ISSUE TITLE
API questions

### ISSUE THREAD

**Issue by doomedramen** [galaxyFlavorsGenerators/galaxyFlavorGenerator#7]
@martenson 
from the toolshed I need to get the tool name (the one that uniquely identifies the tool), is this what repo_name referes to?

example: https://testtoolshed.g2.bx.psu.edu/api/tools?q=kraken

(p.s) has something changed recently, a couple of weeks ago searching for kraken returned no results

**Comment by martenson** [comment]
you have found a bug, it is being tracked here: https://github.com/galaxyproject/galaxy/issues/562

I regenerated the index yesterday, that is why there are new results.

**Comment by doomedramen** [comment]
Ok, then can I query the repo api by repo-name and repo-owner ?

**Comment by martenson** [comment]
Exactly. IF you use `owner` and `name` params you should get the one you want

**Comment by doomedramen** [comment]
Awesome, cheers. New version of the flavours generator is going live
tomorrow btw

**Comment by martenson** [comment]
awesomesauce
